paymentsheet/src/main/java/com/stripe/android/link/ui/signup/SignUpViewModel.kt

```diff
@@ -206,7 +192,9 @@ internal class SignUpViewModel @Inject constructor(
     }
 
     private fun onAccountFetched(linkAccount: LinkAccount?) {
-        if (linkAccount?.isVerified == true) {
+        if (linkAccount?.completedSignup == true) {
+            navigateAndClearStack(LinkScreen.PaymentMethod)
+        } else if (linkAccount?.isVerified == true) {
             navigateAndClearStack(LinkScreen.Wallet)
```

Comment:
Previously, we'd open the wallet screen only to redirect to the payment method screen.